### PR TITLE
typeof with an open generic type with 2 type params

### LINKED ISSUE #11257: statement not clear
I am not sure what this means. is it possible to restate this differently and clearly:
Types with more than one type parameter must have the appropriate number of commas in the specification.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8e46579c-383b-1049-fee9-59c27328f7ba
* Version Independent ID: e9a776dd-e2b4-a2b2-6db4-4de7a4a0baeb
* Content: [typeof - C# Reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/typeof)
* Content Source: [docs/csharp/language-reference/keywords/typeof.md](https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/keywords/typeof.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by rpetrusha**
## typeof with an open generic type with 2 type params

Fixes #11257

### CODE REVIEW

File: docs/csharp/language-reference/keywords/typeof.md [hunk 1]
```diff
@@ -29,7 +29,13 @@ System.Type type = i.GetType();
 
 The `typeof` operator cannot be overloaded.
 
-The `typeof` operator can also be used on open generic types. Types with more than one type parameter must have the appropriate number of commas in the specification. The following example shows how to determine whether the return type of a method is a generic <xref:System.Collections.Generic.IEnumerable%601>. <xref:System.Type.GetInterface%2A?displayProperty=nameWithType> will return `null` if the return type is not an <xref:System.Collections.Generic.IEnumerable%601> generic type.
+The `typeof` operator can also be used on open generic types. Types with more than one type parameter must have the appropriate number of commas in the specification. For example:
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/typeof.md]:
```suggestion
The `typeof` operator can also be used on open generic types. Types with more than one type parameter must have the appropriate number of commas in the specification. For example, the <xref:System.Collections.Generic.Dictionary%602?displayProperty=nameWIthType> has two type arguments, so you use one comma:
```

**Review by BillWagner — APPROVED** [review]
This LGTM @rpetrusha 

I made one clarifying suggestion to show that the `Dictionary` class has two type arguments. It's up to you if you want to accept that.

**Review by BillWagner — APPROVED** [review]
This LGTM @rpetrusha 

I made one suggestion that anticipates potential reader questions.

**Comment by rpetrusha** [comment]
Thanks, @BillWagner. Very good suggestion.

**Comment by rpetrusha** [comment]
Closing and reopening to trigger new CLA check.

**Pull request merged**